### PR TITLE
(Feature) Toaster on contract download

### LINKED ISSUE #270: Show toast notification when a file with contracts is downloaded
Problem:
We need to download the file to user's computer because he may forget to do this. Now we do it without asking user's permission and not notifying him. It may be considered as unfriendly action by a user.

Solution:
Present a toast notification in the right top corner with a text "A file with contracts and metadata downloaded on your computer"

### PR DESCRIPTION
**Pull request by fernandomg**
Closes #270 

- Added a toaster informing the user that the contract was properly downloaded on the PC.
- Moved the Contract Download option after the last transaction, to avoid "Loading" overlay and "Toaster" overlapping.

alert-on-contract-download

### CODE REVIEW

**Comment by fernandomg** [comment]
Depends on https://github.com/oraclesorg/ico-wizard/pull/336

File: src/components/invest/index.js [hunk 1]
```diff
@@ -19,6 +20,11 @@ export class Invest extends React.Component {
       state.loading = true;
       state.pristineTokenInput = true;
       this.state = state;
+
+      this.alertOptions = {
```
Comment (15chrjef) [inline src/components/invest/index.js]: extract to variable in constants file and import

File: src/components/stepFour/index.js [hunk 2]
```diff
@@ -14,13 +14,15 @@ import { DisplayTextArea } from '../Common/DisplayTextArea'
...
 import { copy } from '../../utils/copy';
+import AlertContainer from 'react-alert'
 const { PUBLISH } = NAVIGATION_STEPS
 
 export class stepFour extends stepTwo {
   constructor(props) {
     super(props);
     let oldState = getOldState(props, defaultState)
     this.state = Object.assign({}, oldState)
+    this.alertOptions = { time: 10000, position: 'top right' }
```
Comment (15chrjef) [inline src/components/stepFour/index.js]: extract and as variable to constant file

File: src/components/stepFour/index.js [hunk 3]
```diff
@@ -497,6 +498,8 @@ export class stepFour extends stepTwo {
                         transferOwnership(web3, this.state.contracts.token.abi, contracts.token.addr, this.state.crowdsale[0].walletAddress, 46699, (err) => {
                           if (err) return this.hideLoader();
                           this.hideLoader();
+                          this.downloadCrowdsaleInfo();
+                          this.showToaster({ message: 'A file with contracts and metadata downloaded on your computer' })
```
Comment (15chrjef) [inline src/components/stepFour/index.js]: extract message to constant

**Pull request merged**